test/format/verify.c

```diff
@@ -57,9 +57,16 @@ table_verify(TABLE *table, void *arg)
     wt_wrap_open_session(conn, &sap, table->track_prefix,
       enable_session_prefetch() ? SESSION_PREFETCH_CFG_ON : NULL, &session);
     ret = session->verify(session, table->uri, "strict");
-    testutil_assert(ret == 0 || ret == EBUSY ||
-      /* FIXME-WT-15413: Verify on follower may return ENOENT if stable uri is missing. */
-      (g.disagg_storage_config && !g.disagg_leader && ret == ENOENT));
+    /*
+     * On followers, verify returns ENOENT if the stable constituent is missing. Before the first
```

Comment:
@mariammojid My understanding is that if an ingest exists without a stable table, that’s still a valid state. In that case, verify should internally check the ingest and simply skip the stable table without raising any errors.

However, if neither of the dhandles exists, we should ideally return, catch, and report the error as a bug.

Reply:
> My understanding is that if an ingest exists without a stable table, that’s still a valid state.

Yep! That’s the case for followers. However, we haven't yet extended `verify` to handle the verification of the ingest constituent of a follower. So if a follower is missing local metadata info or it's stable table constituent is missing, it will return `ENOENT`. The plan is to implement verification of the ingest constituent of followers as part of [WT-15047](https://jira.mongodb.org/browse/WT-15047).

I get your intention though. In future, ideally we would skip stable verification in this scenario, verify the ingest constituent, but still report both statuses (i.e. `ENOENT` on stable, `success` or `failure` on ingest).